Ghost/ghost.gd

```diff
@@ -1,6 +1,7 @@
 extends CharacterBody2D
 
-
+func _ready():
+	%AnimatedSprite2D.play("idle")
```

Comment:
Instead of calling the animation "idle", the animation could be called "walk" instead.